apps/portal/pages/index.tsx

```diff
@@ -6,7 +6,12 @@ import { ACMButton } from '@acmutd/acm-ui';
 // commit with above line active and below line uncommented, use below line for testing if you want
 // import { ACMButton } from '@acmutd/acm-ui/src';
 
-export default function HomePage() {
+export const getServerSideProps = async (ctx) => {
+  const { profileVisited } = ctx.req.cookies;
+  return { props: { profileVisited: profileVisited ?? null } };
```

Comment:
Couldn't this just be `return { props: { profileVisited } };`?

Reply:
> Couldn't this just be `return { props: { profileVisited } };`?

![image](https://user-images.githubusercontent.com/62899156/194769933-3262b259-6309-4b8f-b3e7-b1de663e4e20.png)

No, apparently it does need to be like this